### PR TITLE
Improve README in ckcp

### PR DESCRIPTION
**Pull request by bnallapeta**
- Some information in ckcp README was obsolete. Updated it.
- Replaced ArgoCD to Argo CD.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ argocd login argocd-server-argocd.apps.127.0.0.1.nip.io:8443
 
 Argo CD web UI is accessible at <https://argocd-server-argocd.apps.127.0.0.1.nip.io:8443/applications>.
 
-GitOps is the preferred approach for deploying the Pipeline Service. The installed instances of ArgoCD can be leveraged for creating organisations in kcp, universal workspaces used by the infrastructure, installing the Tekton controllers and registering the workload clusters.
+GitOps is the preferred approach for deploying the Pipeline Service. The installed instances of Argo CD can be leveraged for creating organisations in kcp, universal workspaces used by the infrastructure, installing the Tekton controllers and registering the workload clusters.
```
Comment (ramessesii2) [inline DEVELOPMENT.md:105]: `organisations` --> kcp uses the term `organization`.

Comment (bnallapeta) [reply DEVELOPMENT.md:105]: what if you want to create more than one organization? :P

Comment (bnallapeta) [reply DEVELOPMENT.md:105]: if you're talking about the 's' and the 'z' - its not a big deal. Just a different form of the same word (US vs UK based English)

Comment (ramessesii2) [reply DEVELOPMENT.md:105]: > if you're talking about the plural form of the word - what if you want to create more than one organization? :P

ah, I meant `organizatoins` ;), missed `s`

**Review by Roming22 — APPROVED** [review]

**Comment by bnallapeta** [comment]
@fgiloux all tests have passed on this. Please merge the PR as I see approvals from Romain and you.

**Pull request merged**